src/verifier/compare_outputs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    echo "Missing argument for name of existing CSV file from verified_output to check against."
+    exit $1
+fi
+
+VERIFIED=verified_output/$1
+UNVERIFIED=output/${2:-analysis_neighborhood_overall_scores.csv}
+
+echo "Comparing ${VERIFIED} to ${UNVERIFIED}"
+
+csvdiff -q id ${VERIFIED} ${UNVERIFIED}
+
+if [ $? -eq 0 ]; then
+    echo "Output matches!"
+else
+    echo "Output mismatch:"
+    csvdiff id --style pretty ${VERIFIED} ${UNVERIFIED}
```

Comment:
I meant still provide the diff but also pass through the exit status.  Maybe in the actual use cases for this thing that wouldn't matter, but if it gets integrated into any kind of scripted or automated process having it exit abnormally on mismatch would be useful, and it seems like the sort of thing one would expect a "verifier" to do.

Summary table might be a better default, actually, if we end up using this on big CSVs (e.g. database table dumps rather than summary statistics), but there are probably pros and cons either way, so the diff is fine.